### ISSUE TITLE
[dev] Hide "Select primary category" when radios for category chosen

### ISSUE THREAD

**Issue by leymannx** [helgatheviking/Radio-Buttons-for-Taxonomies#52]
I checked out the dev branch for WP5 compatibility and radios appear nicely for the chosen taxonomies.

But when I have chosen radios for category I still get the "Select primary category". I'd expect this select should be hidden, as when selecting a category with radios I can only have one. Which makes that "Select primary category" form element needless.

![Screenshot 2019-04-10 10 47 37](https://user-images.githubusercontent.com/3585653/55865511-cfb0fb00-5b7e-11e9-9665-d1cc032d5c97.png)

**Comment by helgatheviking** [comment]
Is "Select primary category" being added by another plugin? I don't believe that's core.

**Comment by leymannx** [comment]
I thought that's core. I have a nearly-vanilla WordPress running. I'll double-check this in an absolutely vanilla WordPress and get back to you.

**Comment by leymannx** [comment]
Ah yeah! You were right. That's Yoast/wpseo. Unfortunately they don't offer an option to switch that off. Maybe radio-buttons-for-taxonomies might offer another option to hide that? 🤔

**Comment by helgatheviking** [comment]
I think that's gonna be way beyond me. It does look like they are adding a filter to the same spot as my plugin. [See here](https://github.com/Yoast/wordpress-seo/blob/1446228e8f3de8babb77e8a308e59c8408e30743/js/src/wp-seo-metabox-category.js#L252).

Somehow you'll have to find a way to remove that filter. 

I tried adding

```
wp.hooks.removeFilter( 'editor.PostTaxonomyType', 'yoast-seo' );
```

to my index.js and rebuilding the script, but it broke the rest of it.

**Comment by leymannx** [comment]
I see. Damn!

**Comment by leymannx** [comment]
I guess we should drop that issue for now, what do you think? It should definitely not be a blocker for a new release.

**Comment by helgatheviking** [comment]
Yes, I agree that it's not a blocking issue here. If they would add a
filter to control whether it displays, I'm ok with adding the code to turn
it off from my plugin. But without a filter, idk if there's anything I can
do about it.

**Comment by helgatheviking** [comment]
Closed by d6b1bb14769a4a15778a15893a3f0148a13c820d
